src/firefox/preferences.js

```diff
@@ -28,8 +28,8 @@ const prefsCommon: FirefoxPreferences = {
   // Disable the prompt for allowing connections.
   'devtools.debugger.prompt-connection': false,
 
-  // Turn off platform logging because it is a lot of info.
-  'extensions.logging.enabled': false,
+  // Allow extensions to log messages on browser's console
+  'extensions.logging.enabled': true,
```

Comment:
"extensions.logging.enabled" is not the right about:config preference (as it enables verbose logs for some Firefox internals related to the extensions, e.g. the AddonManager is going to log more verbosely).

The one that we want is **devtools.browserconsole.contentMessages**, as also mentioned in #1702 issue summary, which makes the Browser Console to list messages coming from Firefox child processes (by default only the logs emitted from the main Firefox process are going to be visible in the Browser Console).